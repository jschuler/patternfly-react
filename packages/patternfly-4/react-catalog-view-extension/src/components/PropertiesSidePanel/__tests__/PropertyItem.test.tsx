// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { PropertyItem } from '../PropertyItem';

describe('PropertyItem test', () => {
  it('PropertyItem should match snapshot', () => {
    const view = shallow(
      <PropertyItem
        className={"''"}
				label={'string'}
				value={null}
      />);
    expect(view).toMatchSnapshot();
  });
});
