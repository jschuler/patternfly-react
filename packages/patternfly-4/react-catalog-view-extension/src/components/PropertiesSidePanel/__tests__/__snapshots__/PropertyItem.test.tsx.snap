// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyItem test PropertyItem should match snapshot 1`] = `
<div
  className="properties-side-panel-pf-property ''"
>
  <Text
    className="properties-side-panel-pf-property-label"
    component="h5"
  >
    string
  </Text>
  <div
    className="properties-side-panel-pf-property-value"
  />
</div>
`;
